posts/2025-03-03-Project-Goals-Feb-Update.md

```diff
@@ -0,0 +1,1710 @@
+  * [ ] `-Zno-jump-tables`, considered an ABI-modifying flag
+  * [ ] `-Zbinary_dep_depinfo=y` -- basically ready to stabilize (@tmandry asked "Do you want toolchain runtimes (libstd, compiler-rt) in your dep info? In my experience this features does 90% of what I want it to do, but removing the inclusion of runtimes is the only question I have before stabilizing", but we don't understand this point yet, as they were not present in the meeting).
+* stable rustdoc: PR rust-lang/rust#134531 is under review, should land soon, then next step will be for RFL folks to try it out.
+* Clippy configuration: no progress, discussed some new options and posted in thread, very minimal work required here.
+* rebuild libcore: @davidtwco wasn't present so no update, but we know progress is being made
+
+### Publicizing this work
+
+We discussed briefly how to better get the word out about this collaboration. Some points:
+
+* @Darksonn will be speaking at Rust Nation
+* We could author a Rust-lang blog post, perhaps once the language items are done done done?
+* LWN article might be an option
+
+### general-regs-only
+
+We discussed the possibility of a flag to avoid use of floating point registers, no firm conclusion yet reached.
+
+<!-- this comment helps to convince the markdown parser to do the right thing -->
+
+</blockquote>
+
+
+<!-- this comment helps to convince the markdown parser to do the right thing -->
+
+<a href="https://github.com/rust-lang/rust-project-goals/issues/116#issuecomment-2672338598">Comment by @nikomatsakis posted on 2025-02-20:</a><br>
+
+<blockquote>
+
+<!-- this comment helps to convince the markdown parser to do the right thing -->
+
+Updates from our [2025-02-12 meeting](https://hackmd.io/AkNBW942SoacLayPXHthCg):
+
+Given the recent controversy about Rust usage in the Kernel, the RFL group wrote up a [policy document explainer](https://rust-for-linux.com/rust-kernel-policy) to explain the policy, and there was a [write-up on LWN](https://lwn.net/SubscriberLink/1007921/9020dbb12585d48f/).
+
+Regarding arbitary self types and coerce pointee, we are waiting on rust-lang/rust#136764 and rust-lang/rust#136776. The former is on lang team FCP. The latter has received approval from lang team and is awaiting further impl work by @BoxyUwU.
+
+@ojeda is looking into how to manage dependency information and configure no-std externally.
+
+@GuillaumeGomez's impl of rustdoc features has landed and we are waiting on RFL to experiment with it.
+
+@davidtwco's team at ARM has authored a document regarding a blessed way to build-std and are collecting feedback.
+
+@wesleywiser is preparing a PR to add `-Zdwarf-version` to help advance compiler flags.
+
+There is an annoying issue related to `cfg(no_fp_fmt_parse)`, which is no longer used by RFL but which remains in an older branch of the kernel (6.12, LTS). 
```

Comment:
```suggestion
There is an annoying issue related to `cfg(no_global_oom_handling)`, which is no longer used by RFL but which remains in an older branch of the kernel (6.12, LTS). 
```

Reply:
Thanks. I'll fix it in the goal update as well.